fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/cursors/FlatMapPipelinedCursor.java

```diff
@@ -326,17 +326,17 @@ public byte[] toBytes() {
 
             if (innerContinuation.isEnd()) {
                 // This was the last of the inner cursor. Take continuation from outer after it.
-                builder.setOuterContinuation(ByteString.copyFrom(outerResult.getContinuation().toBytes()));
+                builder.setOuterContinuation(ZeroCopyByteString.wrap(outerResult.getContinuation().toBytes()));
```

Comment:
I don't see a harm here in using a `ZeroCopyByteString`, but we should also consider using the `RecordCursorContinuation::toByteString` method on continuations in situations like these. The `toByteString` method is newer, and so not all of the continuations are updated, but for continuations that implement it, it should skip a copy (potentially more than one).